### PR TITLE
Merge in Template V1.6.

### PR DESCRIPTION
**Pull request by apeltzer**
Many thanks to contributing to nf-core/eager!

Please fill in the appropriate checklist below (delete whatever is not relevant). These are the most common things requested on pull requests (PRs).

## PR checklist
 - [x] This comment contains a description of changes (with reason)
 - [ ] If you've fixed a bug or added code that should be tested, add tests!
 - [ ] If necessary, also make a PR on the [nf-core/eager branch on the nf-core/test-datasets repo]( https://github.com/nf-core/test-datasets/pull/new/nf-core/eager)
 - [x] Ensure the test suite passes (`nextflow run . -profile test,docker`).
 - [x] Make sure your code lints (`nf-core lint .`).
 - [x] Documentation in `docs` is updated
 - [x] `CHANGELOG.md` is updated
 - [ ] `README.md` is updated

**Learn more about contributing:** https://github.com/nf-core/eager/tree/master/.github/CONTRIBUTING.md

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,6 +1,7 @@
 FROM nfcore/base
-LABEL description="Docker image containing all requirements for nf-core/eager pipeline"
+LABEL authors="Alexander Peltzer, Stephen Clayton, James A Fellows-Yates" \
```
Comment (jfy133) [inline Dockerfile]: My name is: James A. Fellows Yates

Also Maxime should be included now.

File: README.md [hunk 2]
```diff
@@ -1,25 +1,30 @@
 # nf-core/eager
-**A fully reproducible aDNA analysis pipeline.**
 
-[![Build Status](https://travis-ci.org/nf-core/eager.svg?branch=master)](https://travis-ci.org/nf-core/eager)
+**A fully reproducible and modern ancient DNA pipeline in Nextflow and with cloud support.**.
```
Comment (jfy133) [inline README.md]: Typo: should be: `ancient and modern DNA pipeline`

File: README.md [hunk 3]
```diff
@@ -1,25 +1,30 @@
...
 
 [![install with bioconda](https://img.shields.io/badge/install%20with-bioconda-brightgreen.svg)](http://bioconda.github.io/)
 [![Docker](https://img.shields.io/docker/automated/nfcore/eager.svg)](https://hub.docker.com/r/nfcore/eager)
-![Singularity Container available](
-https://img.shields.io/badge/singularity-available-7E4C74.svg)
 
-### Introduction
-The pipeline is built using [Nextflow](https://www.nextflow.io), a workflow tool to run tasks across multiple compute infrastructures in a very portable manner. It comes with docker / singularity containers making installation trivial and results highly reproducible.
+## Introduction
+The pipeline is built using [Nextflow](https://www.nextflow.io), a workflow tool to run tasks across multiple compute infrastructures in a very portable manner. It comes with docker containers making installation trivial and results highly reproducible.
```
Comment (jfy133) [inline README.md]: Does it not also include singularity? Even if it's a docker -> singularity conversion?

File: README.md [hunk 4]
```diff
@@ -1,25 +1,30 @@
...
 
 
-### Documentation
+## Documentation
 The nf-core/eager pipeline comes with documentation about the pipeline, found in the `docs/` directory:
 
-1. [Installation](docs/installation.md)
+1. [Installation](https://nf-co.re/usage/installation)
```
Comment (jfy133) [inline README.md]: Maybe that should be `Nextflow Installation`

File: README.md [hunk 5]
```diff
@@ -1,25 +1,30 @@
...
 2. Pipeline configuration
-    * [Local installation](docs/configuration/local.md)
-    * [Adding your own system](docs/configuration/adding_your_own.md)
+    * [Local installation](https://nf-co.re/usage/local_installation)
```
Comment (jfy133) [inline README.md]: And `Pipeline installation`

File: README.md [hunk 6]
```diff
@@ -1,25 +1,30 @@
...
+    * [Adding your own system config](https://nf-co.re/usage/adding_own_config)
+    * [Reference genomes](https://nf-co.re/usage/reference_genomes)
 3. [Running the pipeline](docs/usage.md)
 4. [Output and how to interpret the results](docs/output.md)
-5. [Troubleshooting](docs/troubleshooting.md)
+5. [Troubleshooting](https://nf-co.re/usage/troubleshooting)
```
Comment (jfy133) [inline README.md]: I've added stuff to this page in the EAGER repo. I guess thisstuff  would need to be shifted to a new section if there will now be a central nf-core troubleshooting page?

File: README.md [hunk 7]
```diff
@@ -1,25 +1,30 @@
...
+
+<!-- TODO nf-core: Add a brief overview of what the pipeline does and how it works -->
+
+## Credits
+nf-core/eager was originally written by Alexander Peltzer, Stephen Clayton, James A Fellows-Yates.
```
Comment (jfy133) [inline README.md]: Add Maxime

File: assets/email_template.html [hunk 8]
```diff
@@ -5,7 +5,7 @@
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1">
 
-  <meta name="description" content="nf-core/eager: A fully reproducible aDNA analysis pipeline.">
+  <meta name="description" content="nf-core/eager: A fully reproducible and modern ancient DNA pipeline in Nextflow and with cloud support.">
```
Comment (jfy133) [inline assets/email_template.html]: Typo: ancient and modern

File: docs/README.md [hunk 9]
```diff
@@ -2,10 +2,11 @@
...
-    * [Reference genomes](configuration/reference_genomes.md)
+1. [Installation](https://nf-co.re/usage/installation)
+2. Pipeline configuration
+    * [Local installation](https://nf-co.re/usage/local_installation)
+    * [Adding your own system config](https://nf-co.re/usage/adding_own_config)
+    * [Reference genomes](https://nf-co.re/usage/reference_genomes)
+3. [Running the pipeline](usage.md)
 4. [Output and how to interpret the results](output.md)
-5. [Troubleshooting](troubleshooting.md)
+5. [Troubleshooting](https://nf-co.re/usage/troubleshooting)
```
Comment (jfy133) [inline docs/README.md]: See above

File: README.md (hunk 3 above)
Comment (apeltzer) [reply README.md]: True actually, I think thats a mistake here 👍

File: nextflow.config [hunk 10]
```diff
@@ -3,85 +3,94 @@
...
 dag {
   enabled = true
-  file = "${params.tracedir}/nf-core/eager_dag.svg"
+  file = "${params.tracedir}/pipeline_dag.svg"
 }
 
 manifest {
   name = 'nf-core/eager'
-  author = 'Project author name (use a comma to separate multiple names).'
+  author = 'Alexander Peltzer, Stephen Clayton, James A Fellows-Yates'
```
Comment (jfy133) [inline nextflow.config]: Add maxime and fix my name

File: nextflow.config [hunk 11]
```diff
@@ -3,85 +3,94 @@
...
   homePage = 'https://github.com/nf-core/eager'
-  description = 'A fully reproducible aDNA analysis pipeline.'
+  description = 'A fully reproducible and modern ancient DNA pipeline in Nextflow and with cloud support.'
```
Comment (jfy133) [inline nextflow.config]: ancient and modern DNA

File: README.md (hunk 6 above)
Comment (apeltzer) [reply README.md]: Actually we should do that yes 👍

**Comment by jfy133** [comment]
I can't modify the files for some reason so sorry for all the comments.

File: README.md (hunk 6 above)
Comment (jfy133) [reply README.md]: I'll make a backup of it now and then you can accept the PR once minor things fixed.

**Comment by apeltzer** [comment]
No worries - just addressed all your mentioned issues. Troubleshooting is back, all other changes have been done and Maxime is on the authors list now 👍 Welcome @maxibor ;-)

**Comment by apeltzer** [comment]
This just looks fancier than ever before:

```
INFO: ===========
 LINTING RESULTS
=================
 104 tests passed   0 tests had warnings   0 tests failed
The command "nf-core lint ${TRAVIS_BUILD_DIR}" exited with 0.
126.57s$ nextflow run ${TRAVIS_BUILD_DIR} -profile test,docker --pairedEnd --saveReference
N E X T F L O W  ~  version 0.32.0
Launching `/home/travis/build/nf-core/eager/main.nf` [boring_minsky] - revision: 1d1dd2b1ab
----------------------------------------------------
                                        ,--./,-.
        ___     __   __   __   ___     /,-._.--~'
  |\ | |__  __ /  ` /  \ |__) |__         }  {
  | \| |       \__, \__/ |  \ |___     \`-._,-`-,
                                        `._,._,'
  nf-core/eager v2.0.7dev
----------------------------------------------------
Pipeline Name     : nf-core/eager
Pipeline Version  : 2.0.7dev
Run Name          : boring_minsky
Reads             : data/*{1,2}.fastq.gz
Fasta Ref         : https://raw.githubusercontent.com/nf-core/test-datasets/eager2/reference/Mammoth_MT_Krause.fasta
BAM Index Type    : CSI
Data Type         : Paired-End
Skip Collapsing   : No
Skip Trimming     : No
Max Memory        : 6 GB
Max CPUs          : 2
Max Time          : 2d
Output dir        : ./results
Working dir       : /home/travis/build/nf-core/eager/tests/work
Container Engine  : docker
Container         : nfcore/eager:dev
Current home      : /home/travis
Current user      : travis
Current path      : /home/travis/build/nf-core/eager/tests
Script dir        : /home/travis/build/nf-core/eager
Config Profile    : test,docker
Config Description: Minimal test dataset to check pipeline function
----------------------------------------------------
[warm up] executor > local
[e3/5c2d3f] Submitted process > makeFastaIndex (Mammoth_MT_Krause.fasta)
[bb/c844bd] Submitted process > makeSeqDict (Mammoth_MT_Krause.fasta)
[5a/3591a9] Submitted process > makeBWAIndex (Mammoth_MT_Krause.fasta)
[63/29a32b] Submitted process > fastqc (JK2782_TGGCCGATCAACGA_L008)
[ae/239e99] Submitted process > fastqc (JK2785_TGGCCGATCAACGA_L008)
[c1/81d732] Submitted process > output_documentation (1)
[6d/5a9f1a] Submitted process > adapter_removal (JK2782_TGGCCGATCAACGA_L008)
[dd/bc54c9] Submitted process > adapter_removal (JK2785_TGGCCGATCAACGA_L008)
[44/f5ad86] Submitted process > fastqc_after_clipping (JK2782_TGGCCGATCAACGA_L008_R1_001)
[fb/090e0f] Submitted process > bwa (JK2782_TGGCCGATCAACGA_L008_R1_001)
[58/7c4390] Submitted process > bwa (JK2785_TGGCCGATCAACGA_L008_R1_001)
[ed/cc6ced] Submitted process > fastqc_after_clipping (JK2785_TGGCCGATCAACGA_L008_R1_001)
[83/fb42b7] Submitted process > samtools_idxstats (JK2782_TGGCCGATCAACGA_L008_R1_001.sorted)
[3a/09d4a0] Submitted process > samtools_filter (JK2782_TGGCCGATCAACGA_L008_R1_001.sorted.bam)
[44/50937a] Submitted process > damageprofiler (JK2782_TGGCCGATCAACGA_L008_R1_001.sorted)
[05/fb2840] Submitted process > samtools_idxstats (JK2785_TGGCCGATCAACGA_L008_R1_001.sorted)
[f9/737607] Submitted process > samtools_filter (JK2785_TGGCCGATCAACGA_L008_R1_001.sorted.bam)
[f6/7da438] Submitted process > damageprofiler (JK2785_TGGCCGATCAACGA_L008_R1_001.sorted)
[93/19c7ac] Submitted process > dedup (JK2782_TGGCCGATCAACGA_L008_R1_001.sorted.bam.filtered)
[54/eb93e5] Submitted process > qualimap (JK2782_TGGCCGATCAACGA_L008_R1_001.sorted.bam.filtered)
[94/d4b534] Submitted process > get_software_versions (1)
[53/8cd6be] Submitted process > get_software_versions (2)
[a9/ba65ba] Submitted process > dedup (JK2785_TGGCCGATCAACGA_L008_R1_001.sorted.bam.filtered)
[7e/947a50] Submitted process > qualimap (JK2785_TGGCCGATCAACGA_L008_R1_001.sorted.bam.filtered)
[bd/ed314b] Submitted process > preseq (JK2782_TGGCCGATCAACGA_L008_R1_001.sorted.bam.filtered)
[9a/adb2ca] Submitted process > preseq (JK2785_TGGCCGATCAACGA_L008_R1_001.sorted.bam.filtered)
[de/6f9511] Submitted process > multiqc
WARN: [nf-core/eager] Could not attach MultiQC report to summary email
Warning, pipeline completed, but with errored process(es) 
Number of ignored errored process(es) : 0 
Number of successfully ran process(es) : 27 
[nf-core/eager] Pipeline completed successfully
WARN: To render the execution DAG in the required format it is required to install Graphviz -- See http://www.graphviz.org for more info.
The command "nextflow run ${TRAVIS_BUILD_DIR} -profile test,docker --pairedEnd --saveReference" exited with 0.
```

**Pull request merged**